server/lib/s3backup/__init__.py

```diff
@@ -160,9 +159,9 @@ def put_tarball(self, Name=None, Body=None, Size=0, ContentMD5=None, Bucket=None
                             Bucket=self.bucket_name, Key=Key)
                         self.logger.error("Multi-upload to s3 failed:"
                                           " {key},etag doesn't match,"
-                                          " so moving it to ETAG_FAILURE for inspection".
+                                          " so moving it to BACKED-UP-FAILED for inspection".
```

Comment:
BACKUP-FAILED
But this is a somewhat presumptuous message, because `put_tarball` does *not know* what its called will do with the failure. It's better to limit the logging here to what `put_tarball` knows went wrong and leave details like what happens to the tarball to the caller to log for itself (if necessary).